packages/ag-charts-enterprise/src/features/annotations/annotationsModule.ts

```diff
@@ -136,11 +136,11 @@ export const AnnotationsModule: _ModuleSupport.Module = {
             },
             'arrow-up': {
                 fill: _Theme.PALETTE_UP_FILL,
-                handle: { ...handle },
+                handle: { ...handle, stroke: _Theme.DEFAULT_ANNOTATION_COLOR },
```

Comment:
Ah yeh, the stroke is not in `handle` cos the other annotations take it from their line stroke and pass it into the handle. My bad, sorry!